Assets/SEE/Game/SceneManipulation/GameNodeMarker.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using SEE.GO;
+using SEE.Utils;
+using UnityEngine;
+
+namespace SEE.Game.SceneManipulation
+{
+    public static class GameNodeMarker
+    {
```

Comment:
The method should be named `ToggleMarker` because it does not only add a marker but also remove one if one exists.